test/tests/restrictable/undefined_ids/tests

```diff
@@ -2,7 +2,7 @@
   [block]
     type = 'RunException'
     input = 'undefined_block_kernel.i'
-    expect_err = "the following block ids do not exist on the mesh: 10"
+    expect_err = "the following block (ids) do not exist on the mesh: 10 (10)"
```

Comment:
I think it just adds confusion if the name is the ID and gets duplicated in the parenthesis

Reply:
umm so if the name is the id then I just output the id